cypress/integration/diagrams.spec.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+describe('renders diagram', () => {
+  beforeEach(() => {
+    cy.visitTestEditor()
+  })
+
+  it('markmap', () => {
+    cy.codemirrorFill('```markmap\n- pro\n- contra\n```')
+    cy.getMarkdownBody()
+      .find('svg')
```

Comment:
Tbh i'm not happy with just searching for any svg. If I switch markmap with e.g. graphviz, then this test would still be happy. Please use an (additional) more specific selector. But don't add a new CSS class. Use `data-cy` like we do in other tests.

Reply:
`data-cy` added